apps/src/applab/ai.js

```diff
@@ -0,0 +1,70 @@
+import $ from 'jquery';
+import designMode from './designMode';
+
+export default function autogenerateML(modelId) {
+  return new Promise(function(resolve, reject) {
+    $.ajax({
+      url: `/api/v1/ml_models/${modelId}`,
+      method: 'GET'
+    })
+      .then(modelData => {
+        var x = 20;
```

Comment:
Maybe hoist this to a standalone UI-related function, to separate it from the ajax code?